src/EA.Weee.RequestHandlers/AatfEvidence/GetAatfNotesRequestHandler.cs

```diff
@@ -39,16 +39,21 @@ public async Task<List<EvidenceNoteData>> HandleAsync(GetAatfNotesRequest messag
             authorization.EnsureCanAccessExternalArea();
             authorization.EnsureOrganisationAccess(message.OrganisationId);
 
-            //TODO: this only temporary until the compliance filter is added to aatf screens
-            var currentDate = SystemTime.Now;
+            var currentDate = message.CurrentDate.GetValueOrDefault(DateTime.UtcNow);
             var systemSettings = await systemDataDataAccess.Get();
 
             if (systemSettings.UseFixedCurrentDate)
             {
                 currentDate = systemSettings.FixedCurrentDate;
             }
-            
-            var filter = new NoteFilter(currentDate.Year)
+
+            var complianceYear = message.SelectedComplianceYear.GetValueOrDefault(DateTime.UtcNow.Year);
+            if (complianceYear == 0)
+            {
+                complianceYear = DateTime.UtcNow.Year;
+            }
+
+            var filter = new NoteFilter(currentDate.Year, NoteType.EvidenceNote.Value)
```

Comment:
here just pass in the request selected compliance year. and remove the current date property.